docs/usage-guide/topics/ch02-initialization.md

```diff
@@ -1,10 +1,17 @@
 # Initialization and Teardown
-The s2n-tls library must be initialized with `s2n_init()` before calling most library functions. `s2n_init()` MUST NOT be called more than once, even when an application uses multiple threads or processes. s2n attempts to clean up its thread-local memory at thread-exit and all other memory at process-exit. However, this may not work if you are using a thread library other than pthreads or other threads using s2n outlive the thread that called `s2n_init()`. In that case you should call `s2n_cleanup_thread()` from every thread or process created after `s2n_init()`.
 
-> Note: `s2n_cleanup_thread()` is currently considered unstable, meaning the API is subject to change in a future release. To access this API, include `api/unstable/cleanup.h`.
+## Initialization
+The s2n-tls library must be initialized with `s2n_init()` before using the library functions. `s2n_init()` will error if it is called more than once per process.
 
 Initialization can be modified by calling `s2n_crypto_disable_init()` or `s2n_disable_atexit()` before `s2n_init()`.
 
-An application can override s2n-tls’s internal memory management by calling `s2n_mem_set_callbacks` before calling s2n_init.
+An application can override s2n-tls’s internal memory management by calling `s2n_mem_set_callbacks` before calling `s2n_init()`.
 
 If you are trying to use FIPS mode, you must enable FIPS in your libcrypto library (probably by calling `FIPS_mode_set(1)`) before calling `s2n_init()`.
+
+## Teardown
+### Thread-local Memory
+s2n-tls has thread-local memory that it attempts to clean up automatically at thread-exit. This is done using pthread destructors and may not work if you are using a threads library other than pthreads. You can call `s2n_cleanup()` from every thread or process created after `s2n_init()` if you notice thread-local memory leaks.
+
+### Library Cleanup
+A full cleanup and de-initialization of the library can be done by calling `s2n_cleanup_final()`.
```

Comment:
Do we want to explain why this is just "can happen" instead of "must happen"? Idk how useful or clear that would be though.

Reply:
Yeah, there's so many different ways people use s2n-tls that I kind of don't want to elaborate here.

Reply:
I mean I wouldn't go into how you use s2n-tls, I'd more go into what happens if you do or don't do the full cleanup. Like what do I gain from calling that API? Why would I?

Reply:
What about this:
```
A full cleanup and de-initialization of the library can be done by calling `s2n_cleanup_final()`.
s2n-tls allocates some memory at initialization that will be cleaned up by this call.
You may see memory leaks if you do not call this function. 
```
I dunno, I feel like the only people who really care about this API would be people who are constantly loading and unloading s2n in the same process. Or people who care about memory leaks.

Reply:
We don't want to say "you may see memory leaks if you do not call this function", that would mean it really should be called. I was thinking something more like "s2n-tls allocates some memory at initialization that is intended to live for the duration of the process, but can be cleaned up earlier with s2n_cleanup_final. Not calling this method may cause tools like ASAN or valgrind to detect memory leaks, as the memory will still be allocated when the process exits".